index.html

```diff
@@ -183,40 +188,75 @@ <h3>LET'S CONNECT</h3>
           <button type="button">Get my resume</button>
         </div>
         <div class="Language">
-          <div class="lang">
+          <div class="lang flex-row">
             <h2>Languages</h2>
             <i class="fa fa-angle-down fa-2x" aria-hidden="true"></i>
           </div>
           <div class="Lang-box">
             <div class="js-img-box">
-              <img src="img/Ellipse 1.png" alt="" />
               <ul>
-                <li>Javascript</li>
+                <li><img src="img/Ellipse 1.png" alt="" /></li>
+                <li class="L1">Javascript</li>
               </ul>
             </div>
```

Comment:
Your design
![image](https://user-images.githubusercontent.com/78251927/228898951-8a7c841b-7643-45ce-b9d6-16681623edaa.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/228899048-65581045-2aa1-4a70-b290-ad5a30a989a3.png)


According to [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md), we should replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Please kindly change the background color of the items to match the template on Figma 👍